themes/default/content/blog/zephyr-code-placement-and-stacks/index.md

```diff
@@ -0,0 +1,140 @@
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - scott-lowe
+    - christian-nunciato
+    - aaron-kao
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - development-environment
+    - cloud-engineering
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+This is the first in a series of blog posts where we'll explore how a fictional company---Zephyr Archaeotech Emporium---uses Pulumi to manage their online retail store. In this first post, we'll explore a couple common questions that users ask when working with Pulumi; specifically, where should I store my Pulumi code? And how do I support multiple environments with Pulumi? This post will provide some guidance and recommendations around these topics, using Zephyr and their online store as the use case.
+
+<!--more-->
+
+Here are links to all the blog posts in the series: <!--The names of these posts and the order in which we publish them is mutable-->
+
+**Zephyr: Code Placement and Stacks** (this post)
+
+Zephyr: Structuring Their Pulumi Projects
+
+Zephyr: Moving Infrastructure State Between Projects
+
+Zephyr: Tying Stacks Together with Stack References
+
+Zephyr: Evolving the Application
+
+Zephyr: Adding Pulumi Deployments
+
+Zephyr: Refactoring for Reuse
+
+Zephyr: Local Testing with Pulumi
+
+## What is Zephyr?
+
+Zephyr is short for Zephyr Archaeotech Emporium, the fictional company in our scenario. <!--Aaron, need details on the story/background here-->
+
+## The application: Zephyr's online store
+
+Zephyr's application is the second generation of their online store. Their first generation was a monolith that they manually deployed. It was sufficient in Zephyr's early days, but as word spread and volume increased, Zephyr found it difficult to make small, incremental changes to the store, and as a result the velocity of their development efforts slowed. In preparation for the next phase of their growth---driven by robust sales of arcane artifacts---Zephyr's team evaluated a number of different architectures that would enable them to increase development velocity and give them greater flexibility in scaling different aspects of their online store. In the end, they settled on a containerized architecture deployed to Kubernetes because some of the existing team was already familiar with these technologies.
```

Comment:
>word spread and volume increased, Zephyr found it difficult to make small, incremental changes to the store

It feels like we skip over this too quickly. Maybe mention something like the team was so busy managing infrastructure and deployments by hand that they didn't have time to work on actual features (assuming that's kind of what you had in mind)?

Reply:
I tried to address this in 35635a95, let me know if you feel it needs more.